### PR TITLE
Add Sensei title and style onboarding screen

### PR DESCRIPTION
**Pull request by mikeyarce**
### Overview:
senseisetup

### Notes: 
- When the screen size get's smaller, the image becomes centered in the container as it gets smaller.  Another option would be to crop the image but I think centering it looks better
- The Site Title and the Style are both saved as `state` here - do we need to do anything further?  I'm assuming we can use the state later in the onboarding to set the `NewSiteBlogDetails`.   
- Right now they are just saved as colors like `green`.  I'm assuming we'll change this to match whatever option we need to set from the Sensei Theme when it's done.
- When you click on a different style, the image changes to show the style, but the Title does not change.  I felt like that would be maybe a big task for a small change so I opted to not do it.
- Right now, the `initialSiteTitle` is set to `test` - is that intended?  It looks like it's coming from the `ONBOARD_STORE`

#### Testing Instructions

- Start Calypso
- Go to http://calypso.localhost:3000/setup/senseiSetup?flow=sensei
- Change styles, change site title
- Click Continue and make sure you are able to select a domain and you see suggestions with your Site Title.

cc @dadish 

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/style.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+			margin-top: 0;
+		}
+	}
+
+	.form-radio-with-thumbnail {
+		display: grid;
+
+		.form-radio,
+		.form-radio__label {
+			// display: none;
```
Comment (dadish) [inline client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/style.scss]: Let's remove this if we don't need it.

**Review by dadish — COMMENTED** [review]
Superb work @mikeyarce ! 🚀 

> - When the screen size get's smaller, the image becomes centered in the container as it gets smaller. Another option would be to crop the image but I think centering it looks better

It looks great this way.

> - The Site Title and the Style are both saved as state here - do we need to do anything further? I'm assuming we can use the state later in the onboarding to set the NewSiteBlogDetails.

The site title gets set into the `ONBOARD_STORE` once the user presses **Continue**.

We will also set the styles into the `ONBOARD_STORE` once the user presses **Continue**. But for now it's good as it is.

> - Right now they are just saved as colors like green. I'm assuming we'll change this to match whatever option we need to set from the Sensei Theme when it's done.

That's right.

> - When you click on a different style, the image changes to show the style, but the Title does not change. I felt like that would be maybe a big task for a small change so I opted to not do it.

Agree. It will have to involve building those images as components. That's a good amount of work. We would also need to include the fonts that Sensei Theme uses. I created a separate [ticket](https://github.com/Automattic/sensei-pro/issues/1881) this.

> - Right now, the initialSiteTitle is set to test - is that intended? It looks like it's coming from the ONBOARD_STORE

The `ONBOARD_STORE` is cached by calypso in your local storage. If you want to start fresh you'll need to delete some elements from your storage.

screen-shot-2022-10-31-at-11 22 46

File: client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/style.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+$sensei-color: #43af99;
+
+.sensei-setup {
+	.progress-bar__progress {
+		background-color: $sensei-color;
+	}
+
+	.css-1l7urtg-Container {
+		height: auto;
+	}
```
Comment (dadish) [inline client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/style.scss]: Just as I mentioned in another PR. You cannot rely on this class. It's auto generated and will not be the same in production builds.

**Comment by mikeyarce** [comment]
Thanks for the feedback, @dadish !!

I've addressed the items in your feedback as well as updated the designs with new images that will show at different breakpoints.  Still waiting on more images for the other colors so for now we just have the new images for the main style.

sensei-breakpoints

**Review by dadish — APPROVED** [review]

**Pull request merged**